File: src/main/java/com/leijendary/spring/boot/template/core/config/properties/DataSourceReadonlyProperties.java
package com.leijendary.spring.boot.template.core.config.properties;

import com.zaxxer.hikari.HikariConfig;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "spring.datasource.readonly")
public class DataSourceReadonlyProperties extends HikariConfig {
}

File: src/main/kotlin/com/leijendary/config/properties/DataSourceReadOnlyProperties.kt
package com.leijendary.config.properties

import com.zaxxer.hikari.HikariConfig
import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties("spring.datasource.read-only", ignoreUnknownFields = false)
class DataSourceReadOnlyProperties : HikariConfig()
